### PR TITLE
Updated the media library header to expose the external services

### PR DESCRIPTION
**Pull request by marcinbot**
screen shot 2017-09-15 at 17 00 05

CC @ice9js @beaucollins

### CODE REVIEW

File: client/my-sites/media-library/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		}
+
+		return (
+			<CompactCard className="media-library__footer" style={ { width: this.state.width + 'px' } }>
+				<div className="media-library__footer-buttons">
+					<Button onClick={ this.onCancel }>{ translate( 'Cancel' ) }</Button>
+				</div>
+				<div className="media-library__footer-count">
+					<Count count={ selectedItems.length } />
+					<span className="media-library__footer-count-label">{ translate( 'Selected' ) }</span>
```
Comment (a8ci18n) [inline client/my-sites/media-library/footer.jsx]: Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'selected' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/magnificient-media-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/media-library/upload-controls.jsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+				<div>
+					<SectionHeader
+						className="media-library__upload-controls-header"
+						label={ translate( 'Upload from' ) } />
+					<Card className="media-library__upload-controls-options">
+						{ this.renderFileItem() }
+						{ this.renderUploadViaUrl() }
+						<CompactCard className="media-library__upload-controls-option" onClick={ this.onExternalServiceClick }>
+							<SocialLogo icon="google" size={ 18 } />
+							<span className="media-library__upload-controls-option-name">{ translate( 'Google Photos' ) }</span>
```
Comment (a8ci18n) [inline client/my-sites/media-library/upload-controls.jsx]: Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Photos from Google' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/magnificient-media-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/media-library/upload-controls.jsx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+		if ( ! userCan( 'upload_files', site ) ) {
+			return null;
+		}
+
+		return (
+			<div className="media-library__upload-controls-container">
+				<div>
+					<SectionHeader
+						className="media-library__upload-controls-header"
+						label={ translate( 'Upload from' ) } />
```
Comment (a8ci18n) [inline client/my-sites/media-library/upload-controls.jsx]: Hi! I've found a possible matching string that has already been translated **104** times:
`translate( 'Upload' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/media-library/upload-controls.jsx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+	}
+
+	renderFileItem() {
+		const { translate } = this.props;
+
+		return (
+			<form ref="itemForm">
+				<CompactCard className="media-library__upload-controls-option">
+					<Gridicon icon="computer" size={ 18 } />
+					<span className="media-library__upload-controls-option-name">{ translate( 'Computer' ) }</span>
```
Comment (a8ci18n) [inline client/my-sites/media-library/upload-controls.jsx]: Hi! I've found a possible matching string that has already been translated **96** times:
`translate( 'From Computer' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by beaucollins** [comment]
I believe we are required by Google to use the multi-color icon and use the label "Photos by Google" not "Google Photos".

**Pull request closed without merging**